src/me/StevenLawson/TotalFreedomMod/Commands/Command_suspend.java

```diff
@@ -0,0 +1,104 @@
+package me.StevenLawson.TotalFreedomMod.Commands;
+
+import me.StevenLawson.TotalFreedomMod.TFM_AdminList;
+import me.StevenLawson.TotalFreedomMod.TFM_Ban;
+import me.StevenLawson.TotalFreedomMod.TFM_BanManager;
+import me.StevenLawson.TotalFreedomMod.TFM_PlayerList;
+import me.StevenLawson.TotalFreedomMod.TFM_Util;
+import org.bukkit.ChatColor;
+import org.bukkit.GameMode;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+import org.bukkit.scheduler.BukkitRunnable;
+import org.bukkit.util.Vector;
+
+@CommandPermissions(level = AdminLevel.SENIOR, source = SourceType.ONLY_CONSOLE, blockHostConsole = true)
+@CommandParameters(description = "Used to suspend admins", usage = "/<command> <playername>")
+public class Command_suspend extends TFM_Command
+{
+    @Override
+    public boolean run(final CommandSender sender, Player sender_p, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
+    {
+        if (args.length != 1)
+        {
+            return false;
+        }
+
+        final Player player = getPlayer(args[0]);
+
+        if (player == null)
+        {
+            sender.sendMessage(TFM_Command.PLAYER_NOT_FOUND);
+            return true;
+        }
+
+        TFM_Util.adminAction(sender.getName(), "Suspending: " + player.getName(), true);
+        TFM_Util.bcastMsg(player.getName() + " will be suspended!", ChatColor.RED);
+
+        final String ip = player.getAddress().getAddress().getHostAddress().trim();
+
+        // remove from superadmin
+        if (TFM_AdminList.isSuperAdmin(player))
+        {
+            TFM_Util.adminAction(sender.getName(), "Suspending " + player.getName(), true);
+            TFM_AdminList.removeSuperadmin(player);
+        } else {
+            TFM_Util.adminAction(sender.getName(), "Tried to suspend " + player.getName());
+        }
+
+        // remove from whitelist
+        player.setWhitelisted(false);
```

Comment:
I see no reason to remove them from the whitelist... Again, Suspending not dooming.